src/main/app/case/answers/possibleAnswers.ts

```diff
@@ -44,7 +45,19 @@ export const omitUnreachableAnswers = (caseState: Partial<Case>, steps: Step[]):
   pick(caseState, getAllPossibleAnswers(caseState, steps));
 
 export const getUnreachableAnswersAsNull = (userCase: Partial<Case>): Partial<Case> => {
-  const possibleAnswers = getAllPossibleAnswers(userCase, stepsWithContent);
+  const possibleAnswers = [
+    ...getAllPossibleAnswers(userCase, stepsWithContent),
```

Comment:
```suggestion
    ...getAllPossibleAnswers(userCase, stepsWithContentApplicant1),
```

Reply:
To be refactored once Seb's branch is merged